docs/core/runtime-config/threading.md

```diff
@@ -94,6 +94,44 @@ Project file:
 </Project>
 ```
 
+## Windows thread pool
+
+- For projects on Windows, configures whether the thread management is delegated to the Windows thread pool.
+- If you omit this setting or the platform is not Windows, the managed (portable) thread pool is used instead.
+- Only applications published with Native AOT on Windows use the Windows thread pool by default, for which you can opt to use the managed thread pool instead by disabling the config setting.
+
```

Comment:
Should add a couple more notes, maybe something like:
- The Windows thread pool may perform better in some cases, such as in cases where the minimum number of threads is configured to a high value, or when the Windows thread pool is already being heavily used by the app. There may also be cases where the .NET thread pool performs better, such as in heavy I/O handling on larger machines. It is advisable to check performance metrics when changing this config setting.
- Some APIs are not supported when using the Windows thread pool, such as [`ThreadPool.SetMinThreads`](https://learn.microsoft.com/en-us/dotnet/api/system.threading.threadpool.setminthreads), [`ThreadPool.SetMaxThreads`](https://learn.microsoft.com/en-us/dotnet/api/system.threading.threadpool.setmaxthreads), and [`ThreadPool.BindHandle`](https://learn.microsoft.com/en-us/dotnet/api/system.threading.threadpool.bindhandle). Thread pool config settings for minimum and maximum threads are also not effective. An alternative to `ThreadPool.BindHandle` is the [`ThreadPoolBoundHandle`](https://learn.microsoft.com/en-us/dotnet/api/system.threading.threadpoolboundhandle) class.